Untitled.py

```diff
@@ -0,0 +1,53 @@
+
+
+
+def sheet_data(sheet):
+
+    out = list()
+
+    sheetIndex = 0
+    for X in range(1, s.nrows):
+        learner = s.row_values(X)
+        print(learner)
+        average=sum(learner)
```

Comment:
the first two columns in the spreadsheet contain student's number and name - we can't sum up all values in a row
also, it won't be an average